cypress/locators/carousel/locators.js

```diff
@@ -4,7 +4,8 @@ export const TRANSITION_SELECT = 'select[name="transition"]';
 
 // component preview locators
 export const CAROUSEL = '.carbon-carousel__transition';
-export const SLIDE = 'div[data-element="visible-slide"]';
+export const CLASSIC_SLIDE = 'div[data-element="visible-slide"]';
+export const CAROUSEL_CONTENT = '.carbon-carousel__content';
```

Comment:
I asked @DlgSHi about this when distinguishing between classic and default. As classic will be disappearing at some point in the near future then default-related elements should not be prefixed with `DEFAULT_` as we'd have to change all of those in addition to removing `CLASSIC_` ones. I'll make the change to the element for the default slide though as part of implementation of your suggested solution to avoid the need for a `for` loop 👍

Reply:
ok, so maybe use just `SLIDE` instead `DEFAULT_SLIDE` or `CAROUSEL_CONTENT`
This locator indicates exactly this element, so it's important to have a proper name for it :)

Reply:
Amended using your suggested solution 👍